src/Orckestra.Composer.Cart/Factory/Order/OrderHistoryViewModelFactory.cs

```diff
@@ -235,7 +236,7 @@ protected virtual IEnumerable<OrderHistoryPageViewModel> GetPages(OrderHistoryPa
 
             orderInfo.OrderStatus = GetOrderStatusDisplayName(rawOrder, param);
             orderInfo.OrderStatusRaw = rawOrder.OrderStatus;
-            orderInfo.IsOrderEditable = IsOrderEditable(param, rawOrder);
+            orderInfo.IsOrderEditable = param.OrderEdititngInfos.ContainsKey(Guid.Parse(rawOrder.Id)) ? param.OrderEdititngInfos[Guid.Parse(rawOrder.Id)] : false;
```

Comment:
Can be simplified as
`param.OrderEdititngInfos.ContainsKey(Guid.Parse(rawOrder.Id)) && param.OrderEdititngInfos[Guid.Parse(rawOrder.Id)];`